preprocessing/DataCleanser.py

```diff
@@ -3,14 +3,17 @@
 
 def dropNulls(df_covid):
     """
-    Removes any rows that contain a null value. We print out the abstract description here because we know there are
-    null value in there. We made the abstract a None type whenever no abstract was present in a JSON file.
+    Removes any rows that contain a null value. We currently create a string 'no abstract provided' if the abstract
+    is not present, so this method is to account for potential mishandled abstract entries.  For 'body_text'
+    entries that are empty strings, we convert them to NaN and then drop them.
 
     Should be completed before running other methods to reduce the number of rows examined.
 
     :param df_covid: Dataframe of covid data.
     :return: Data frame without any null values.
     """
+    df_covid['body_text'].replace("", np.nan, inplace=True)
+    df_covid['body_text'].replace(r'^\s+$', np.nan, regex=True)
```

Comment:
Line 16 also checks for an empty space I think. I don't think line 15 is needed. Also, I think it may be better to replace with None instead of Nan. Idk if Nan is considered the same as null.

Reply:
Will remove line 15
NaN is considered the same as None

https://pandas.pydata.org/pandas-docs/stable/user_guide/missing_data.html#missing-data